### PR TITLE
Add alerts to users

### PR DESCRIPTION
**Pull request by AbhiPrasad**
Adds `alertsViewed` to the User Schema

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -31,7 +31,11 @@ const UserSchema = new mongoose.Schema({
     type: String,
     required: true
   },
-  activities: [String]
+  activities: [String],
+  alertsViewed: [{
+    name: String,
```
Comment (theRoughCode) [inline models/User.js]: we could just store an array of alert ID's, no?  Do we need the name and description?

**Review by theRoughCode — CHANGES_REQUESTED** [review]
Have a question on user alerts list

File: models/User.js (hunk 1 above)
Comment (AbhiPrasad) [reply models/User.js]: Yeah you're right, I'll do it like that

**Review by theRoughCode — APPROVED** [review]
👍

**Pull request merged**